get_s3.go

```diff
@@ -119,25 +119,25 @@ func (g *S3Getter) Get(dst string, u *url.URL) error {
 	lastMarker := ""
 	hasMore := true
 	for hasMore {
-		req := &s3.ListObjectsInput{
+		req := &s3.ListObjectsV2Input{
 			Bucket: aws.String(bucket),
 			Prefix: aws.String(path),
 		}
 		if lastMarker != "" {
-			req.Marker = aws.String(lastMarker)
+			req.Delimiter = aws.String(lastMarker)
```

Comment:
I don't have as much experience with AWS SDK as Graham does but I'd think that delimiters are not typically used in the context of pagination. Graham mentioned earlier that we could use the native paginators, as documented at https://aws.github.io/aws-sdk-go-v2/docs/making-requests/#using-paginators

@lebauce have you considered that?

Reply:
@radeksimko Pagination is not used yet if I'm not mistaken. Do we want to implement this in this PR ?

Reply:
@lebauce in this case pagination is making multiple requests to `ListObjectsV2` while `ContinuationToken` is set (was `Marker` in `ListObjects`). In `aws-sdk-go-v2`, AWS has added the `NewListObjectsV2Paginator` that can be used to handle pagination instead of using a `for` loop.